components/common/ConnectWallet/WalletAssetIdentity.vue

```diff
@@ -70,8 +70,11 @@ const logout = async () => {
   walletStore.setDisconnecting(true)
   walletStore.clear()
   if (isEvm) {
-    await disconnectWeb3Modal()
+    await disconnectWeb3Modal().catch(() => {
+      walletStore.setDisconnecting(false)
```

Comment:
there's a `walletStore.setDisconnecting(false)` after the if , no need to call it again , let's just log it , should do the same

```suggestion
    await disconnectWeb3Modal().catch((error) => {
      console.log('[WEB3MODAL::CONNECTION] Failed disconnecting', error)
```

Reply:
make sense. I will make it a warning log.